### PR TITLE
Update how_this_course_will_work.md

### PR DESCRIPTION
**Pull request by xPedroNoronhax**
Making the final message more exciting, adding an exclamation point! I believe removing ' and adding !!! will get the reader more excited to start learning React.

## Because
Making the final message more exciting, adding an exclamation point!
I believe removing ' and adding !!! will get the reader more excited to start learning React.

##This PR
  - A change occurred on line 15 of the how_this_course_will_work.md file

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [ x] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [ x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x ] The `Because` section summarizes the reason for this PR
-   [ x] The `This PR` section has a bullet point list describing the changes in this PR
-   [ x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [ x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [ x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Review by xandora — COMMENTED** [review]
While `reactin'` is barely a word, `reactin` on the other hand, is a non-steroidal anti-inflammatory. 

Joking aside, I think if we're going to keep this text as a heading, (which I'm leaning towards removing or changing now that it's brought up...) we probably want to keep the apostrophe and stick to a single exclamation mark.

**Comment by ManonLef** [comment]
@xPedroNoronhax  are you willing to make the suggested change?

File: react/introduction/how_this_course_will_work.md [hunk 1]
```diff
@@ -12,4 +12,4 @@ As always, we'll preface this by saying that learning a new technology can be a
 
 Now then,
 
-### Let’s start Reactin’
+### Let’s start Reactin!!
```
Comment (ManonLef) [inline react/introduction/how_this_course_will_work.md:15]:
```suggestion
### Let’s start Reactin'!
```

**Review by ManonLef — CHANGES_REQUESTED** [review]
I have left a suggestion as requested by @xandora 
I'm of the same opinion. 

You can change this file locally and commit the change, or commit my suggestion for the same result and re-request a review :)

**Pull request closed without merging**